packages/ui-extensions/src/surfaces/admin/components/Link/examples/RelativeLink.example.ts

```diff
@@ -1,12 +0,0 @@
-import {extension, Link} from '@shopify/ui-extensions/admin';
-
-export default extension('Playground', (root) => {
-  const productLink = root.createComponent(Link, {
-    url: '/admin/products/12345',
```

Comment:
This example is also useful but should be updated to reflect that the urls will be resolved relative to app